### PR TITLE
[INS] Update docs IP Mobility

### PR DESCRIPTION
**Pull request by bene2k1**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

File: compute/instances/how-to/migrate-routed-ips.mdx [hunk 1]
```diff
@@ -21,7 +21,8 @@ Using a routed flexible IP is recommended for all compatible Instance types, as
 
 <Message type="important">
   * The routed IP feature **is incompatible** with legacy Instance types using the bootscript feature.
-  * If you use a manual DHCP configuration on the public NIC, ensure it is compatible with a routed IP setup.
+  * Ensure no static network configuration is in use for your public network interface (usually `ens2` or `eth0`)
+  * Ensure that you are not using the internal private IP of the Instance for internal communications. Instead [attach your Instances to a VPC](/network/vpc/how-to/attach-resources-to-pn/).
```
Comment (SamyOubouaziz) [inline compute/instances/how-to/migrate-routed-ips.mdx:25]:
```suggestion
  * Ensure that you are not using the internal private IP of the Instance for internal communications. Instead, [attach your Instances to a VPC](/network/vpc/how-to/attach-resources-to-pn/).
```

File: faq/instances.mdx [hunk 2]
```diff
@@ -257,4 +257,43 @@ Check the [Instances availability guide](/console/account/reference-content/prod
...
 
-This Region is entirely powered by renewable (hydraulic) energy. It also has an energetic footprint 30-40% lower than traditional data centers, thanks to the fact that it does not require air conditioning. Learn more about [our environmental commitment](https://www.scaleway.com/en/environmental-leadership/).
\ No newline at end of file
+This Region is entirely powered by renewable (hydraulic) energy. It also has an energetic footprint 30-40% lower than traditional data centers, thanks to the fact that it does not require air conditioning. Learn more about [our environmental commitment](https://www.scaleway.com/en/environmental-leadership/).
+
+## Network
+
+## Which Instances support IP Mobility?
+
+All ranges are compatible except for Instances using the “bootscript” feature, which has [reached its end of life (EOL)].
```
Comment (SamyOubouaziz) [inline faq/instances.mdx]: link missing ?

File: faq/instances.mdx [hunk 3]
```diff
@@ -257,4 +257,43 @@ Check the [Instances availability guide](/console/account/reference-content/prod
...
+
+### Can I enable “Routed IP” on my bootscript Instance?
+
+Instances using a bootscript are not compatible with Routed IPs. As the bootscript feature has reached its end of life (EOL), please refer to [this tutorial](/compute/instances/troubleshooting/bootscript-eol/) for migration instructions.
```
Comment (SamyOubouaziz) [inline faq/instances.mdx:270]:
```suggestion
Instances using a bootscript are not compatible with Routed IPs. As the bootscript feature has reached its end of life (EOL), refer to [this tutorial](/compute/instances/troubleshooting/bootscript-eol/) for migration instructions.
```

File: faq/instances.mdx [hunk 4]
```diff
@@ -257,4 +257,43 @@ Check the [Instances availability guide](/console/account/reference-content/prod
...
+
+No, you will receive a new IPv6 after migration.
+
+### Will my Instance retain its private IPv4 after migration?
+
+No, the Scaleway private IP will be removed after the migration.
+
+### Will my Instance retain its VPC private IPv4 after migration?
+
+Yes, your VPC configuration will remain intact after migration to a routed public IPv4.
```
Comment (SamyOubouaziz) [inline faq/instances.mdx:290]:
```suggestion
Yes, your VPC configuration will remain intact after migrating to a routed public IPv4.
```

**Review by nerda-codes — APPROVED** [review]

File: faq/instances.mdx (hunk 2 above)
Comment (bene2k1) [reply faq/instances.mdx]: Yes. fixed.

File: faq/instances.mdx [hunk 5]
```diff
@@ -257,4 +257,43 @@ Check the [Instances availability guide](/console/account/reference-content/prod
...
+Yes, your VPC configuration will remain intact after migrating to a routed public IPv4.
+
+### Can a resource with a routed IP communicate with a resource with a NAT IP?
+
+Yes, they can communicate with each other using their public IPs.
+
+### Are there any limitations on IP autoconfiguration with the routed IP feature?
+
+Currently, additional routed IPv6 addresses do not autoconfigure on CentOS 7, 8, 9, Alma 8, 9, Rocky 8, 9 after migration.
+Note that, both additional routed IPv4 and IPv6 addresses are not autoconfigured post-migration on Ubuntu 20.04 focal. However, the primary IPv6 continues to be configured via SLAAC. These limitations are currently being addressed.
```
Comment (jcirinosclwy) [inline faq/instances.mdx:299]:
```suggestion
Note that both additional routed IPv4 and IPv6 addresses are not autoconfigured post-migration on Ubuntu 20.04 focal. However, the primary IPv6 continues to be configured via SLAAC. These limitations are currently being addressed.
```

File: faq/instances.mdx [hunk 6]
```diff
@@ -257,4 +257,43 @@ Check the [Instances availability guide](/console/account/reference-content/prod
 
 `FR-PAR-2` is our sustainable and environmentally efficient Availability Zone (AZ) in Paris.
 
-This Region is entirely powered by renewable (hydraulic) energy. It also has an energetic footprint 30-40% lower than traditional data centers, thanks to the fact that it does not require air conditioning. Learn more about [our environmental commitment](https://www.scaleway.com/en/environmental-leadership/).
\ No newline at end of file
+This Region is entirely powered by renewable (hydraulic) energy. It also has an energetic footprint 30-40% lower than traditional data centers, thanks to the fact that it does not require air conditioning. Learn more about [our environmental commitment](https://www.scaleway.com/en/environmental-leadership/).
+
+## Network
+
+## Which Instances support IP Mobility?
```
Comment (bene2k1) [inline faq/instances.mdx:264]:
```suggestion
### Which Instances support IP Mobility?
```

**Review by jcirinosclwy — APPROVED** [review]

File: faq/instances.mdx [hunk 7]
```diff
@@ -257,4 +257,43 @@ Check the [Instances availability guide](/console/account/reference-content/prod
...
+Yes, your VPC configuration will remain intact after migration to a routed public IPv4.
+
+### Can a resource with a routed IP communicate with a resource with a NAT IP?
+
+Yes, they can communicate with each other using their public IPs.
+
+### Are there any limitations on IP autoconfiguration with the routed IP feature?
+
+Currently, additional routed IPv6 addresses do not autoconfigure on CentOS 7, 8, 9, Alma 8, 9, Rocky 8, 9 after migration.
+Note that, both additional routed IPv4 and IPv6 addresses are not autoconfigured post-migration on Ubuntu 20.04 focal. However, the primary IPv6 continues to be configured via SLAAC. These limitations are currently being addressed.
```
Comment (SamyOubouaziz) [inline faq/instances.mdx:299]: doubts on "Note that", maybe replace by "Also" ? I can't remember what is the rule with this formulation

**Review by SamyOubouaziz — APPROVED** [review]

**Pull request merged**